kustomizations/apps/data-hub-configs/bigquery-to-opensearch--stg.yaml

```diff
@@ -366,3 +366,47 @@ bigQueryToOpenSearch:
         operationMode: 'update'
         upsert: True
     batchSize: 1000
+
+  - dataPipelineId: bigquery_to_opensearch_data_pipeline_s2_specter_v1_embedding
+    source:
+      bigQuery:
+        projectName: 'elife-data-pipeline'
+        sqlQuery: |-
+          SELECT
+              embeddings_response.paper_id AS doi,
+
+              STRUCT(
+                STRUCT(
+                  'model-apis:specter_v1' AS model_id,
+                  embeddings_response.embedding AS vector,
+                  embeddings_response.imported_timestamp AS data_hub_imported_timestamp
+                ) AS specter_embedding_v1
+              ) AS s2
+          FROM `elife-data-pipeline.{ENV}.semantic_scholar_specter_v1_embeddings_web_api_response` AS embeddings_response
+          ORDER BY paper_id
+    fieldNamesFor:
+      id: doi
+      timestamp: s2.specter_embedding_v1.data_hub_imported_timestamp
+    state:
+      initialState:
+        startTimestamp: '2023-11-16+00:00'
```

Comment:
Would today be the expected start time? I just wanted to double check.

Reply:
This is loading data from the BigQuery table that we only started populating 16 Nov (yesterday's today). That is why I put that date in there. Although any date before that would also work in terms of config.